js/battle.js

```diff
@@ -553,7 +556,7 @@ var Pokemon = (function () {
 			if (plaintext) {
 				name += ' (' + this.species + ')';
 			} else {
-				name += ' <small>(' + this.species + ')</small>';
+				name = '<span class="pokemon" title="' +  this.species + '">' + name + ' <small>(' + this.species + ')</small>' + '</span>';
```

Comment:
I did another check before finishing this, and from what I can tell there's two locations that won't be affected by the battle log; tooltips and the icon mouse-over text [here](https://github.com/Zarel/Pokemon-Showdown-Client/blob/master/js/battle.js#L1371). It's not a big problem, but I rather not make a special-case for multiple locations for the same thing, so unless there's an objection to the check staying in `getFullName`, that's what I'll probably end up doing ...